web/mockServiceWorker.js

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (0.34.0).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const INTEGRITY_CHECKSUM = 'f0a916b13c8acc2b526a03a6d26df85f'
```

Comment:
This feels like something that should be treated carefully? Maybe in an environment variable or something

Reply:
Maybe, it's just for local dev and was autogenerated so I'm too fussed about it